library/styleguideComponents/Wrapper/index.js

```diff
@@ -66,8 +64,22 @@ const yellowTheme = {
     },
 };
 
+const orangeDarkTheme = {
+    rainbow: {
+        palette: {
+            brand: '#80deea',
+            success: '#b9f6ca',
+            error: '#ff5252',
+            background: {
+                primary: '#242424',
+                secondary: '#333333',
+            },
+        },
+    },
+};
+
 export default function Wrapper(props) {
-    const { children } = props;
+    const { children, preview } = props;
```

Comment:
I really don't remember but I believe that preview prop here does not exist and is not needed.